manifests/cpp.yml

```diff
@@ -137,6 +137,7 @@ tests/:
       Test_Dsm_Manual_Checkpoint_Intra_Process: missing_feature
   parametric/:
     test_config_consistency.py:
+      Test_Config_DDTraceAgentURL: missing_feature
```

Comment:
Do we need to set this to missing_feature? Can we just get this test to pass in this PR OR leave a descriptive message on why this test is failing (ex: `missing_feauture (cpp does not support uds)`

Reply:
You don't necessarily need to give a reason for `missing_feature`, but it may help later on, so if you have one that is not as obvious as "*we didn't started yet to work on this*" , feel free to add it.

Reply:
@mabdinur setting it to missing_feature as of now because the parametric client apps need to be updated to send dd_trace_agent_url